src/__tests__/actions-test.ts

```diff
@@ -151,10 +149,8 @@ describe("actions", () => {
 
     it("dispatches failure on no response", async () => {
       const dispatch = stub();
-      const fetchMock = stub().returns(new Promise<any>((resolve, reject) => {
-        reject({ message: "test error" });
-      }));
-      fetch = fetchMock;
+      fetchMock
+        .mock(url, () => { throw { message: "test error" }; });
```

Comment:
This took forever to find. It also seems difficult to throw an error (and have no response) but this is how to do it.